### PR TITLE
Unit tests for secret-management APIs

### PR DESCRIPTION
**Pull request by SiniWilson**
### Description of the Change

Unit tests for secret-management APIs

### CODE REVIEW

File: core/jazz_acl/package.json [hunk 1]
```diff
@@ -18,8 +19,9 @@
     "chai": "^3.5.0",
     "chai-as-promised": "^7.1.1",
     "sinon": "^7.2.2",
+    "nyc": "^13.3.0",
```
Comment (raghits) [inline core/jazz_acl/package.json]: Can this just be a devDependency?

File: core/jazz_t-vault/test/test.js [hunk 2]
```diff
@@ -15,13 +15,3519 @@
...
+      .catch((err) => {
+        expect(err).to.include({
+          errorType: 'inputError',
+          message: 'Input cannot be empty'
+        });
+      });
+    done();
+  });
+
+  it('should reject while validating create safe input params with out event.body', (done) => {
```
Comment (raghits) [inline core/jazz_t-vault/test/test.js]: nitpick - with out is a single word `without` - several instances.

File: core/jazz_t-vault/test/test.js [hunk 3]
```diff
@@ -15,13 +15,3519 @@
...
+    done();
+  });
+
+  it('should reject while validating path params with out safename', (done) => {
+    event.path = { "name": "test" };
+    validations.validateSafeInput(event)
+      .catch((err) => {
+        expect(err).to.include({
+          errorType: 'inputError',
+          message: 'Following field(s) are required in path- safename'
```
Comment (raghits) [inline core/jazz_t-vault/test/test.js]: Can we keep this message and format consistent? `Following field(s) are required in path - ` Adding space before and after `-`

File: core/jazz_t-vault/test/test.js [hunk 4]
```diff
@@ -15,13 +15,3519 @@
...
+    done();
+  });
+
+  it('should reject while validating query params with out rolename', (done) => {
+    event.query = { "name": "test" };
+    validations.validateGetRoleInSafeInput(event)
+      .catch((err) => {
+        expect(err).to.include({
+          errorType: 'inputError',
+          message: 'Following field(s) are required in query- rolename'
```
Comment (raghits) [inline core/jazz_t-vault/test/test.js]: Inconsistent format, see comment above.

File: core/jazz_acl/package.json (hunk 1 above)
Comment (SiniWilson) [reply core/jazz_acl/package.json]: Yes. Its a devdependency

File: core/jazz_t-vault/test/test.js (hunk 2 above)
Comment (SiniWilson) [reply core/jazz_t-vault/test/test.js]: Done

File: core/jazz_t-vault/test/test.js (hunk 3 above)
Comment (SiniWilson) [reply core/jazz_t-vault/test/test.js]: Done

File: core/jazz_t-vault/test/test.js (hunk 4 above)
Comment (SiniWilson) [reply core/jazz_t-vault/test/test.js]: Done

**Pull request merged**